website/untp-v1.jsonld

```diff
@@ -1,37 +1,86 @@
 {
     "@context": {
         "@version": 1.1,
-        "@vocab": "https://uncefact.github.io/issuer-dependent#",
+        "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#",
         "id": "@id",
         "type": "@type",
-        "UNTPDigitalProductPassportCredential": {
-            "@id": "https://uncefact.github.io/spec-untp/docs/specification/DigitalProductPassport",
-            "@context": {}
+
+        "schema.org": "https://schema.org/",
+        "uncefact-vocab": "https://vocabulary.uncefact.org/",
+        "untp-spec": "https://uncefact.github.io/spec-untp/docs/specification/",
+        "untp-vocab": "https://test.uncefact.org/vocabulary/untp/",
+
+        "ConformityAssessment": {
+          "@id": "untp-spec:ConformityCredential#conformityassessment",
+          "@context": {
+            "@protected": true,
```

Comment:
Why protected?

Reply:
This was inspired from the [citizenship vocab](https://w3c-ccg.github.io/citizenship-vocab/contexts/citizenship-v2.jsonld) where objects are protected. I suppose the reasoning is because we want to avoid these terms being redefined by another context. If you could explain to me why these shouldn't be or how it could be problematic I'm interested in hearing that perspective as well!